tdm-be/src/controller/traitment.ts

```diff
@@ -17,11 +11,19 @@ import {
     HTTP_PRECONDITION_REQUIRED,
 } from '~/lib/http';
 import logger from '~/lib/logger';
+import type { Processing } from '~/model/ProcessingModel';
 import { createProcessing, findProcessing, updateProcessing } from '~/model/ProcessingModel';
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏I thought types were more isolated

Usually, there was a blank line at least before the `import type`...

Reply:
import/order miss understand the import with ts-paths